pallets/fragments/src/lib.rs

```diff
@@ -442,8 +442,12 @@ pub mod pallet {
 			// create vault account
 			// we need an existential amount deposit to be able to create the vault account
 			let vault = Self::get_vault_id(hash);
-			let min_balance = <T as pallet_assets::Config>::Currency::minimum_balance();
-			<T as pallet_assets::Config>::Currency::reserve(&vault, min_balance)?;
+			let min_balance =
```

Comment:
this is unrelated no?
Also @aang114 fixed it and this would also not work iirc?

Reply:
Yes, it's unrelated. I think it came up merging locally my branch with devel.

Reply:
Weird if was merged should not show though 🤔

Reply:
Weird, agreed. Trying to figure out how to remove these two files from this PR (which are already on devel).